tests/settings.py

```diff
@@ -7,6 +7,8 @@
 
 SECRET_KEY = "django-insecure-2%2hup1&(r-6(88=)txivii25r^o%4-&00u5vgs#n93r#t8+y0"
 
+DEBUG = True
```

Comment:
I think you'd always want to test in 'production' mode. So `DEBUG=False`. As that is the configuration you'd like your app to work.

Or is this with a specific reason?

Reply:
Not for a specific reason, it was just for debugging. I will fix it.